packages/chat/index.ts

```diff
@@ -8,4 +8,4 @@
  */
 
 import './components/test-input/test-input.js';
-import './components/feedback/feedback.js';
+import '../feedback/components/feedback/feedback.js';
```

Comment:
We can probably remove this line as `feedback` is it's own package and not part of the `chat` package

Reply:
Removed